app/blueprints/assessments/routes.py

```diff
@@ -134,6 +137,21 @@
 )
 
 
+def run_with_context(func, *args, **kwargs):
+    with current_app.app_context():
+        return func(*args, **kwargs)
+
+
+class ContextThreadPoolExecutor(concurrent.futures.ThreadPoolExecutor):
```

Comment:
I think in utils we have this ContextThreadPoolExecutor can you check is it can be used

Reply:
I searched for it but I couldn't find anything?

Reply:
may be check this https://github.com/communitiesuk/funding-service-design-utils/blob/main/fsd_utils/sqs_scheduler/context_aware_executor.py

Reply:
Ooo nice - I've updated to use that instead 👍🏽